client/src/Header.jsx

```diff
@@ -0,0 +1,20 @@
+import './index.css';
+
+export default function Header() {
```

Comment:
Good start! For this project, this is okay! However, if you want to challenge yourself, how would we make this reusable?